substra/__init__.py

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from substra.__version__ import __version__
 from substra.sdk import Client, exceptions
+from substra.__version__ import __version__
```

Comment:
Minor: whenever you change something and then undo the change, you should try to undo the change completely and not just undo the functionality change. Here for example we have 2 additional files in the diff that shouldn't be here at all (setup.py and substra/__init__.py) because the changes you made neither change the functionnality or fix/improve formatting. One handy solution to revert all changes made to a file is to check it out from the master branch:

```sh
git checkout master -- substra/__init__.py
git checkout master -- setup.py
```

Reply:
Yes, I should have done that, I'll do it!